docs/install/any-kubernetes-cluster.md

```diff
@@ -23,15 +23,22 @@ showlandingtoc: "false"
 You can install Knative by applying YAML files using the `kubectl` CLI.
 You can install the Serving component, Eventing component, or both on your cluster.
 
+## Recommended System Requirement
```

Comment:
```suggestion
## Recommended system requirements
```

Reply:
I don't think we need to say recommended? Not sure if this was part of the discussion I missed though